src/store/state.ts

```diff
@@ -1,39 +1,43 @@
-import { EntityStatus, CorpTypeCd, FilingStatus } from '@/enums'
-import { FilingDataIF } from '@/interfaces'
+import { StateIF } from '@/interfaces'
 
-export default {
+const state: StateIF = {
   // tombstone data
-  keycloakRoles: [] as Array<string>,
-  authRoles: [] as Array<string>,
-  username: null as string,
-  currentDate: null as string, // 'today' as YYYY-MM-DD
+  keycloakRoles: [],
```

Comment:
I had attempted typing in the old code but it was never carried throughout the code. Now it works correctly (in class syntax files).

You can mouse over any of these properties to see their type.